resources/META-INF/scala-plugin-common.xml

```diff
@@ -1156,6 +1156,10 @@
                          displayName="Multiple arg lists in annotation constructor" groupPath="Scala" groupName="General"
                          shortName="MultipleArgListsInAnnotation" level="WARNING"
                          enabledByDefault="true" language="Scala"/>
+      <localInspection implementationClass="org.jetbrains.plugins.dotty.codeInspection.deprecated.WithTypeInspection"
+                       displayName="Deprecated with type" groupPath="Scala" groupName="Other"
+                       shortName="DottyDeprecatedWith" level="WARNING"
```

Comment:
You need to add `DottyDeprecatedWith.html` file to `resources/inspectionDescriptions`